src/Twig/IiifOpenGraphExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace eLife\Journal\Twig;
+
+use eLife\ApiSdk\Model\Image;
+use eLife\Journal\Helper\CreatesIiifUri;
+use Twig_Extension;
+use Twig_Function;
+use function min;
+
+final class IiifOpenGraphExtension extends Twig_Extension
+{
+    use CreatesIiifUri;
+
+    public function getFunctions()
+    {
+        return [
+            new Twig_Function('iiif_og_image', [$this, 'createOpenGraphImage'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    public function createOpenGraphImage(Image $image) : string
+    {
+        if ($image->getWidth() > $image->getHeight()) {
+            $width = min($image->getWidth(), 2000);
```

Comment:
`2000` should be a constant?

Reply:
Will makes it clear that the two `2000` cannot be change independently (or that they can, if it is the case)